content/en/docs/tasks/security/authorization/authz-ingress/index.md

```diff
@@ -31,143 +26,390 @@ through the Istio ingress gateway with this command:
+for more information. Update the ingress gateway to set `externalTrafficPolicy: Local` to preserve the
+original client source IP on the ingress gateway using the following command:
+
+{{< text bash >}}
+$ kubectl patch svc istio-ingressgateway -n istio-system -p '{"spec":{"externalTrafficPolicy":"Local"}}'
+{{< /text >}}
+
+{{< /tab >}}
+
+{{< tab name="HTTP/HTTPS Load Balancer" category-value="http" >}}
+
+If you are using an HTTP/HTTPS external load balancer (AWS ALB, GCP ), it can put the original client IP address in the X-Forwarded-For header.  Istio can extract the client IP address from this header with some configuration.  See [Configuring Gateway Network Topology](/docs/ops/configuration/traffic-management/network-topologies/). Quick example if using a single load balancer in front of Kubernetes:
+
+{{< text yaml >}}
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  meshConfig:
+    accessLogEncoding: JSON
+    accessLogFile: /dev/stdout
+    defaultConfig:
+      gatewayTopology:
+        numTrustedProxies: 1
+{{< /text >}}
+
+{{< /tab >}}
+
+{{< /tabset >}}
+
+For reference, here are the types of load balancers created by Istio with a `LoadBalancer` service on popular managed Kubernetes environments:
+
+|Cloud Provider | Load Balancer Name            | Load Balancer Type
+----------------|-------------------------------|-------------------
+|AWS EKS        | Classic Elastic Load Balancer | TCP Proxy
+|GCP GKE        | TCP/UDP Network Load Balancer | Network
+|Azure AKS      | Azure Load Balancer           | Network
+|DO DOKS        | Load Balancer                 | Network
+
+{{< tip >}}
+You can instruct AWS EKS to create a Network Load Balancer when you install Istio by using a `serviceAnnotation` like below:
+
+{{< text yaml >}}
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  meshConfig:
+    accessLogEncoding: JSON
+    accessLogFile: /dev/stdout
+  components:
+    ingressGateways:
+    - enabled: true
+      k8s:
+        hpaSpec:
+          maxReplicas: 10
+          minReplicas: 5
+        serviceAnnotations:
+          service.beta.kubernetes.io/aws-load-balancer-type: "nlb"
+{{< /text >}}
+
+{{< /tip >}}
+
 ## IP-based allow list and deny list
 
-1. The following command creates the authorization policy, `ingress-policy`, for
+**`ipBlocks` vs `remoteIpBlocks`:** If you are using the X-Forwarded-For HTTP header or the Proxy Protocol to determine the original client IP address, then you should use `remoteIpBlocks` in your `AuthorizationPolicy`. If you are using `externalTrafficPolicy: Local`, then you should use `ipBlocks` in your `AuthorizationPolicy`.
```

Comment:
In the `TCP/UDP Proxy Load Balancer` tab in the previous section, neither `X-Forwarded-For` nor `externalTrafficPolicy` is mentioned. Which one (ipBlocks or remoteIpBlocks) to use in that case?

This paragraph should also reference back to the previous section to make it clear that this decision is related to that information.

Reply:
I added a table here to make that more clear.